x-pack/plugins/apm/public/components/app/Correlations/SignificantTermsTable.tsx

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import {
+  EuiBadge,
+  EuiIcon,
+  EuiToolTip,
+  EuiLoadingChart,
+  EuiLink,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { isEmpty } from 'lodash';
+import { useHistory } from 'react-router-dom';
+import { asPercent, asInteger } from '../../../../common/utils/formatters';
+import { APIReturnType } from '../../../services/rest/createCallApmApi';
+import { FETCH_STATUS } from '../../../hooks/useFetcher';
+import { createHref } from '../../shared/Links/url_helpers';
+
+type CorrelationsApiResponse =
+  | APIReturnType<'/api/apm/correlations/failed_transactions', 'GET'>
+  | APIReturnType<'/api/apm/correlations/slow_transactions', 'GET'>;
+
+type SignificantTerm = NonNullable<
+  CorrelationsApiResponse
+>['significantTerms'][0];
+
+const TableRow = styled.tr`
+  border-bottom: 1px solid #aaa;
+  border-top: 1px solid #aaa;
+  padding: 5px;
+
+  &:hover {
+    background: #eee;
+  }
+`;
+
+const TableCell = styled.td`
+  padding: 10px;
+
+  // first column
+  &:first-child {
+    width: 1px;
+    white-space: nowrap;
+  }
+`;
+
+interface Props<T> {
+  significantTerms?: T[];
+  status: FETCH_STATUS;
+  setSelectedSignificantTerm: (term: T | null) => void;
+}
+
+export function SignificantTermsTable<T extends SignificantTerm>({
+  significantTerms,
+  status,
+  setSelectedSignificantTerm,
+}: Props<T>) {
+  return (
+    <table>
```

Comment:
I know this is just a PoC but would it be any harder to use an `EuiBasicTable`? Seems like it would be less or equal work.

Reply:
I started with `EuiBasicTable` but couldn't get event handlers to work (specifically `onMouseEnter` and `onMouseLeave`. 
I'll take a look again.